### ISSUE TITLE
Feature Request: Image slideshow from public image repository (Bing, etc.)

### ISSUE THREAD

**Issue by kombuking** [Reflect-OS/firmware#10]
Many users (such as myself) intend to use ReflectOS to run a smart calendar on a screen, rather than a mirror.

For this, it would be visually appealing for the plain black background to be replaced by random images from online sources, such as Bing Images, UnSplash, etc.

Background images would sit behind all Section content and text.

Images would need to randomise/change at regular intervals (Example, 1 hour, 6 hour, 12 hour, 24 hour)

Basically a clone of the following MagicMirror module:
https://github.com/delightedCrow/WallberryTheme
